core/utils.py

```diff
@@ -232,6 +232,24 @@ def dict_from_yaml(yaml_str: str) -> Dict[str, Any]:
         raise InvalidInputException(e)
 
 
+def yaml_from_dict(dictionary: Dict[str, Any], width: int = 80) -> Any:
+    """Gets the YAML representation of a dict.
+
+    Args:
+        dictionary: dict. Dictionary for conversion into yaml.
+        width: int. Width for the yaml representation, default value
+            is set to be of 80.
+
+    Returns:
+        str. Converted yaml of the passed dictionary.
+    """
+    # The type ignore is needed, because typestubs define the return type
+    # of 'safe_dump' as 'Any' which is wrong.
+    return yaml.dump( # type: ignore[no-any-return]
+           dictionary, allow_unicode=True, width=width
+    )
```

Comment:
```suggestion
    # The type ignore is needed, because typestubs define the return type
    # of 'dump' as 'Any' which is wrong.
    return yaml.dump( # type: ignore[no-any-return]
       dictionary, allow_unicode=True, width=width
    )
```

Reply:
yes done